From 40e419d7632b66e1e5db24becf50bb0d805b096d Mon Sep 17 00:00:00 2001
From: Philippe Coval <philippe.coval@osg.samsung.com>
Date: Mon, 4 Dec 2017 11:26:39 +0100
Subject: [PATCH] resource: Remove DirectPairing json file

oic_svr_db_client_directpairing.json is not more used
since DirectPairing was removed

Bug: https://jira.iotivity.org/browse/IOT-2338
Change-Id: I2131730b1a9344bb5d422722f04da53e82bcc3ba
Signed-off-by: Philippe Coval <philippe.coval@osg.samsung.com>
(cherry picked from commit ddd4244120ce60fbe53310a81fd9d6920fed8b09)
Origin: https://gerrit.iotivity.org/gerrit/#/c/23621/
---
 .../examples/oic_svr_db_client_directpairing.json  | 76 ----------------------
 1 file changed, 76 deletions(-)
 delete mode 100644 resource/examples/oic_svr_db_client_directpairing.json

diff --git a/resource/examples/oic_svr_db_client_directpairing.json b/resource/examples/oic_svr_db_client_directpairing.json
deleted file mode 100644
index 40b9a8b01..000000000
--- a/resource/examples/oic_svr_db_client_directpairing.json
+++ /dev/null
@@ -1,76 +0,0 @@
-{
-    "acl": {
-        "aclist": {
-            "aces": [
-                {
-                    "subjectuuid": "*",
-                    "resources": [
-                        {
-                            "href": "/oic/res",
-                            "rel": "",
-                            "rt": ["oic.wk.res"],
-                            "if": ["oic.if.ll"]
-                        },
-                        {
-                            "href": "/oic/d",
-                            "rel": "",
-                            "rt": ["oic.wk.d"],
-                            "if": ["oic.if.baseline", "oic.if.r"]
-                        },
-                        {
-                            "href": "/oic/p",
-                            "rel": "",
-                            "rt": ["oic.wk.p"],
-                            "if": ["oic.if.baseline", "oic.if.r"]
-                        }
-                    ],
-                    "permission": 2
-                },
-                {
-                    "subjectuuid": "*",
-                    "resources": [
-                        {
-                            "href": "/oic/sec/doxm",
-                            "rel": "",
-                            "rt": ["oic.r.doxm"],
-                            "if": ["oic.if.baseline"]
-                        },
-                        {
-                            "href": "/oic/sec/pstat",
-                            "rel": "",
-                            "rt": ["oic.r.pstat"],
-                            "if": ["oic.if.baseline"]
-                        },
-                        {
-                            "href": "/oic/sec/cred",
-                            "rel": "",
-                            "rt": ["oic.r.cred"],
-                            "if": ["oic.if.baseline"]
-                        }
-                    ],
-                    "permission": 6
-                }
-            ]
-        },
-        "rowneruuid" : "64706169-7269-6e67-4465-765555494430"
-    },
-    "pstat": {
-        "dos": {"s": 1, "p": false},
-        "isop": false,
-        "deviceuuid": "64706169-7269-6e67-4465-765555494430",
-        "rowneruuid": "64706169-7269-6e67-4465-765555494430",
-        "cm": 2,
-        "tm": 0,
-        "om": 4,
-        "sm": 4
-        },
-    "doxm": {
-        "oxms": [0],
-        "oxmsel": 0,
-        "sct": 9,
-        "owned": false,
-        "deviceuuid": "64706169-7269-6e67-4465-765555494430",
-        "devowneruuid": "",
-        "rowneruuid": "64706169-7269-6e67-4465-765555494430"
-    }
-}
-- 
2.16.1.windows.1

